oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -598,7 +591,7 @@ private APIGatewayProxyResponseEvent generateAuthRedirect(
         var claimsBuilder =
                 new JWTClaimsSet.Builder()
                         .issuer(configurationService.getOrchestrationClientId())
-                        .audience(configurationService.getLoginURI().toString())
+                        .audience(frontEndPages.baseURI().toString())
```

Comment:
Have we checked this preserved things like trailing slashes? Audience validation is exact, so any changes here could be fatal

Reply:
Good to know. LOGIN_URI and FRONTEND_BASE_URL both always have the value "https://${local.frontend_fqdn}/".  given frontEndPages.baseURI() just calls configurationService.getFrontendBaseURL() an I've replaced all usage of LOGIN_URI with FRONTEND_BASE_URL, in both cases the audience will have a trailing slash.